db/connection.js

```diff
@@ -0,0 +1,11 @@
+var mysql = require('mysql');
+var config = require('./config');
+
+var connection = mysql.createConnection( config );
```

Comment:
You mentioned password/ connection issues at our last stand up. I don't know what the solution was, but will this solve the issue?

Reply:
There are probably a lot of ways to address this, but mine was to make a "config.js" containing this code:

```
module.exports = {
  host: "localhost",
  user: "root",
  password: "{password_test}"
};
```

Because this is not checked into Git, I'll need to pass y'all this file when we get to the point where it needs to be installed in different places.